devel/aspcud/Portfile

```diff
@@ -1,40 +1,40 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem                1.0
-PortGroup           cmake 1.0
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
 PortGroup           boost 1.0
 
-name                aspcud
-version             1.9.1
-revision            3
+github.setup        potassco aspcud 1.9.5 v
 categories          devel math
 platforms           darwin
 maintainers         nomaintainer
-license             GPL-3
+license             MIT
 description         CUDF package dependency solver
 long_description \
     Aspcud is a solver for package dependencies. A package universe and a request \
     to install, remove, or upgrade packages have to be encoded in the CUDF format. \
     Such a CUDF document can then be passed to aspcud along with an optimization \
     criteria to obtain a solution to the given package problem.
 
-homepage            http://potassco.sourceforge.net
-master_sites        sourceforge:project/potassco/aspcud/${version}/
+homepage            https://potassco.org/aspcud/
+github.tarball_from archive
+worksrcdir          ${name}-${version}
```

Comment:
is this line necessary - I think the github PG should set this correctly

Reply:
thx. Last time I wrote a portfile was before the github PG existed 😛